demos/select.scss

```diff
@@ -15,6 +15,7 @@
 //
 
 @import "./common";
+@import "../packages/mdc-button/mdc-button";
```

Comment:
We should be able to remove mdc-list and mdc-menu imports here, right?

Also, we should be able to remove mdc-menu and mdc-list from mdc-select's package.json, which hasn't been touched in this PR.

Reply:
Good call!